src/european_values/llm_evaluation.py

```diff
@@ -0,0 +1,85 @@
+"""LLM evaluation using trained GMM."""
+
+from typing import Any, Dict, Tuple
+
+import joblib
+import numpy as np
+import pandas as pd
+from sklearn.pipeline import Pipeline
+
+
+def load_gmm_pipeline(model_path: str) -> Pipeline:
+    """Load trained GMM pipeline (scaler + model)."""
+    return joblib.load(model_path)
+
+
+def process_responses(responses: np.ndarray) -> np.ndarray:
```

Comment:
Is this function actually necessary? Shouldn't all NaN's already be imputed in the data?